README.md

```diff
@@ -26,10 +26,10 @@ firebase ext:install aloglia-firebase/firestore-algolia-search --project=<your-p
 
 Use this extension to index your Cloud Firestore collection.  The extension is applied and configured on a collection.
 
-This extension listens for changes to the specified collection. If a document is added, updated, or deleted, this
+This extension listens for changes on the specified collection. If a document is added, updated, or deleted, this
```

Comment:
Would recommend the same suggestions here as in the PREINSTALL guide, but only putting them in one place so as to not duplicate/maintain both sets of instructions. (Seems like the README might be a better place than PREINSTALL.)

Reply:
I think we might need to deal with duplicates.  The readme is shown when you visit the Github repo.  The Preinstall is shown in the Extension landing page.